cinema/settings.py

```diff
@@ -147,11 +148,11 @@
 
 EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
 
-EMAIL_HOST = env("EMAIL_HOST", default="")
-EMAIL_PORT = env("EMAIL_PORT", default="")
-EMAIL_USE_TLS = env("EMAIL_USE_TLS", default="")
-EMAIL_HOST_USER = env("EMAIL_HOST_USER", default="")
-EMAIL_HOST_PASSWORD = env("EMAIL_HOST_PASSWORD", default="")
+EMAIL_HOST = env("EMAIL_HOST")
```

Comment:
Why strip out the default values? With no default the app will ask for these values before it can run if no env vars are provided. No need to change anything, but it seems unnecessary